brave/browser-laptop
# Removing clear cache and clear site data options from the history menu

Fix #3093 
There were 3 options to clear history, namely Clear History, Clear Cache, Clear All Cookies and Site Data. These seemed redundant. Removing the 3 and keeping only one option called "Clear Browsing Data". ~~The particular string seems to already exist in history.properties file. Added the same entry with the key "clearBrowsingData" in menu.properties as well.~~

Test Plan:

1. Make sure only one option exists for clearing history, from history menu
 - Open Brave
 - Select the History menu option
 - See that only "Clear Browsing Data" option is available

~~2. Test option name in different supported languages~~
~~- Verify that the option name is correct in different languages~~

PS2: Removed the translation file changes
PS3: Changed "Clear browsing data..." to "Clear Browsing Data...", as requested in the issue

Auditors: @bsclifton
